docs/release_notes/8.0.1-1.md

```diff
@@ -0,0 +1,83 @@
+# Percona Server for MongoDB 8.0.1-1 ({{date.8_0_1}}) 
+
+[Installation](../install/index.md){.md-button}
+[Upgrade](../install/upgrade-from-70.md){.md-button}
+
+We are pleased to announce the release candidate of the new major version of Percona Server for MongoDB 8.0.1-1 – the drop in replacement for MongoDB Community Edition. Percona Server for MongoDB supports MongoDB Community protocols and drivers and enhances it with the [enterprise-grade features for free](../comparison.md), enabling you to effectively meet the requirements of your organization in terms of high-availability, reliability and data security. 
+
+This release of Percona Server for MongoDB includes the improvements and bug fixes of [MongoDB 8.0.0 Community Edition](https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.0---oct-2--2024) and [MongoDB 8.0.1 Community Edition](https://www.mongodb.com/docs/manual/release-notes/8.0/#8.0.1---oct-9--2024)
+
+## Upgrade considerations
+
+Starting with version 8.0, Percona Server for MongoDB uses the upgraded TCMalloc to handle memory requests more efficiently. The use of TCMalloc requires enabling Transparent Huge Pages (THP) and per-CPU caches before you start Percona Server for MongoDB. Read more about TCMalloc in [MongoDB documentation :octicons-external-link-16:](https://www.mongodb.com/docs/manual/administration/tcmalloc-performance/)
```

Comment:
I'm not sure why we bring this as an upgrade consideration?

Reply:
Because this config should be changed after upgrade from 7.0 to 8.0

In MongoDB 8.0 and later, ensure that THP is enabled before mongod starts by creating a service file for your platform's initialization system. If you are using MongoDB 7.0 or earlier - disable THP.

Reply:
Right. @nastena1606  can we add this to the description so it's clearer?

Reply:
Done